src/parser/monk/mistweaver/modules/spells/EnvelopingBreath.tsx

```diff
@@ -38,6 +51,55 @@ class EnvelopingBreath extends Analyzer {
     }
   }
 
+  handleEnvelopingMist(event: CastEvent) {
+    if(this.chijiActive || this.selectedCombatant.hasBuff(SPELLS.INVOKE_YULON_THE_JADE_SERPENT.id)) {
+      this.envsDuringCelestial += 1;
+    }
+  }
+
+  handleEnvelopingBreathCount(event: ApplyBuffEvent) {
+    if(this.chijiActive || this.selectedCombatant.hasBuff(SPELLS.INVOKE_YULON_THE_JADE_SERPENT.id)) {
```

Comment:
You don't need this If statement as you can't get envb procs without these being up soooooooo its unneeded